### PR TITLE
add notes to clarify scan with overlay

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

File: end-user/en/source/pages/migration/scanning.rst [hunk 1]
```diff
@@ -33,24 +33,28 @@ To carry out a scan, go to the ``Migration`` tab:
...
 
-	5. Click ``Next``. Follow the instructions on the UForge AppCenter GUI.
-	6. Download binary locally by clicking ``Download``.
-	7. Copy the binary on the target environment you want to migrate.
-	8. Open a terminal window and login to the target environment.
-	9. Run the scan command on the running target environment to start a deep scan of the system you want to migrate. The binary identifies the packages, files and custom files on the system.
+	7. Download binary locally by clicking ``Download``.
+	8. Copy the binary on the target environment you want to migrate.
+	9. Open a terminal window and login to the target environment.
+	10. Run the scan command on the running target environment to start a scan of the system you want to migrate. Note the ``-o`` option in ghe scan command identifies that you will launch a scan with overlay. The binary identifies the packages, files and custom files on the system.
```
Comment (cinlloc) [inline end-user/en/source/pages/migration/scanning.rst]: typo: ghe > the

File: end-user/en/source/pages/migration/process-overview.rst [hunk 2]
```diff
@@ -5,7 +5,9 @@
 Migration Process Overview
 --------------------------
 
-To migrate a live workload to a new target environment, you first copy the uforge-scan binary to the target environment and launch the binary.  This binary analyzes the live system and sends back a report, known as a "scan" back to UForge AppCenter.  You can then generate a new machine image that is compatible for the target environment.  Once a machine image has been generated, the machine image can then be registered to the cloud environment and used to provision an instance.  This instance will contain the same packages and files as the original running environment.  This is known as “black box” migration.
+To migrate a live workload to a new target environment, you first copy the uforge-scan binary to the target environment and launch the binary.  This binary analyzes the live system and sends back a report. You can select two types of scan: base scan or scan with overlay. A base scan will scan the target environment, but will only return the system packages and configuration known to UForge. A scan with overlay will include an overlay report which details all the extra files, packages or specific configuration.
```
Comment (ammarjohar) [inline end-user/en/source/pages/migration/process-overview.rst]: I'm afraid that the word base in the term "base scan" could be confusing, as it's also used to differentiate between base(first scan)/incremental scans which is applicable for both types with and without overlay. maybe it's better to use only the name "scan" or "scan without overlay".

File: end-user/en/source/pages/migration/whitebox-migration.rst [hunk 3]
```diff
@@ -12,8 +12,8 @@ The goal of white box migration is to change the contents found during the scan
 As part of this transformation process, the scan information is mapped to one or more of the appliance template layers as follows:
 
 	* Native packages that have been analyzed by the scan process and correctly found in one of UForge AppCenter package repositories are added to the ``OS Profile``.
-	* Native packages that have been analyzed by the scan process and NOT found in any of UForge AppCenter's package repositories are added to a ``My Software`` component.
-	* All other files (including configuration files) that are not part of a native package are added to a ``My Software`` component.
+	* Native packages that have been analyzed by the scan process and NOT found in any of UForge AppCenter's package repositories are added to a ``My Software`` component if you performed a scan with overlay.
```
Comment (ammarjohar) [inline end-user/en/source/pages/migration/whitebox-migration.rst]: native packages are added in both cases with/without overlay. I think we don't need to add this here.

File: end-user/en/source/pages/migration/blackbox-migration.rst [hunk 4]
```diff
@@ -7,7 +7,7 @@ Blackbox Migration Process
 
 The goal of black box migration is to reproduce a near identical copy of the currently running workload.  However, there will always be small differences between the two workloads after migration is complete.  When scanning a system the following information is detected:
 
-	* all the files and packages on the system (including configuration information)
+	* all the files and packages on the system (including configuration information). If you have selected a scan without overlay, then extra files and specific configuration information will be detected but will not be included in the report.
```
Comment (ammarjohar) [inline end-user/en/source/pages/migration/blackbox-migration.rst]: in fact the extra files and specific configs are included in the report, but they are not included in the scan object resulting after the scan is done, however I'm not sure if word "report" has the same meaning for us as usharesoft developers and for the users, which could be pointing to the scan object so in this case the docs here are still correct.

File: end-user/en/source/pages/migration/migration-details.rst [hunk 5]
```diff
@@ -52,7 +54,7 @@ The results of this analysis are then sent via HTTPS back to the ``uforge-scan``
 Build the Overlay Archive
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``uforge-scan`` binary retrieves the analysis results from UForge AppCenter. These results include a list of all the packages and files UForge requires.  The ``uforge-scan`` binary builds an overlay archive of all these packages and files.
+The ``uforge-scan`` binary retrieves the analysis results from UForge AppCenter. These results include a list of all the packages and files UForge requires.  The ``uforge-scan`` binary builds an overlay archive of all these packages and files, if the user has performed a scan with overlay.
```
Comment (ammarjohar) [inline end-user/en/source/pages/migration/migration-details.rst]: the if statement here seems to be pointing to both packages and files, while it should be limited only to files

File: end-user/en/source/pages/migration/scanning.rst (hunk 1 above)
Comment (emuus) [inline end-user/en/source/pages/migration/scanning.rst]: ok

File: end-user/en/source/pages/migration/process-overview.rst (hunk 2 above)
Comment (emuus) [inline end-user/en/source/pages/migration/process-overview.rst]: good point

File: end-user/en/source/pages/migration/blackbox-migration.rst (hunk 4 above)
Comment (emuus) [inline end-user/en/source/pages/migration/blackbox-migration.rst]: yes, here "report" is the scan object for the user, so I leave as is

**Pull request merged**